aws-athena/ctas/location-access.sql

```diff
@@ -1,27 +1,28 @@
-CREATE TABLE IF NOT EXISTS location.access
-WITH (
-    FORMAT = 'Parquet',
-    WRITE_COMPRESSION = 'SNAPPY',
-    EXTERNAL_LOCATION = 's3://ccao-athena-ctas-us-east-1/location/access',
```

Comment:
The `FORMAT`, `WRITE_COMPRESSION`, and `EXTERNAL_LOCATION` attributes do not have to be set in the model definition anymore; they are the same for all tables, so we configure them at the model level in `dbt_project.yml` below. On the other hand, we leave `PARTITIONED_BY`, `BUCKETED_BY`, and `BUCKET_COUNT` to be set by individual models, since these values are not identical across all models, or even within schemas.